client/src/app/site/agenda/components/agenda-list/agenda-list.component.html

```diff
@@ -35,7 +37,14 @@
         ></a>
         <div [ngStyle]="{ 'margin-left': item.level * 25 + 'px' }">
             <os-icon-container [icon]="item.closed ? 'check' : null" size="large">
-                {{ item.getListTitle() }}
+                <div>
+                    <div>
+                        {{ item.getListTitle() }}
+                    </div>
+                    <div *ngIf="showSubtitleWithSubmitters" class="subtitle">
+                        {{ getSubtitle(item) }}
```

Comment:
This should be `item.getSubtitle()` Please use generics here, as we do it with titles already. I'll mark other places to change, too.